releasing.md

```diff
@@ -2,6 +2,28 @@
 
 Some preliminary instructions for how to cut a release.
 
+## Authenticate to GCP
+
+If you're new to using GKE or are new to the release team, you'll need to authenticate to GCP first.  [Install the `gcloud` tool](https://cloud.google.com/sdk/gcloud/) and then execute the following commands, substituting your release team account for `your-team-account@kubeflow.org`:
+
+```
+gcloud config set account your-team-account@kubeflow.org
```

Comment:
gcloud config set account <your google account>

Most people won't have kubeflow.org accounts and this makes it seem like they need one.
They don't.

They just need to be added to 
release-team@kubeflow.org